packages/mui-material/src/Select/Select.test.js

```diff
@@ -79,6 +79,88 @@ describe('<Select />', () => {
     );
   });
 
+  it('should support conditional rendering with empty string', () => {
```

Comment:
You could generate all these tests:

```suggestion
  ['', 0, false, undefined, NaN].forEach(value => it(`should support conditional rendering with "${value}"`, () => {
```